whitepapers/0010_ibc_generic_messaging.md

```diff
@@ -0,0 +1,92 @@
+# IBC Bridging
```

Comment:
Should this be titled "IBC Generic Message Emission" so as not to be confused with token bridging via IBC.